src/pages/UserPage/UserPage.tsx

```diff
@@ -168,8 +180,13 @@ const UserPage: FC = () => {
 
   const onActivateSubmit: SubmitHandler<FormValues> = useCallback(
     async (values) => {
-      const payload: UserStatusType = { ...values, institution_user_id }
+      const { notify_user } = values
 
+      const payload: UserStatusType = {
+        ...values,
+        institution_user_id,
+        ...{ notify_user: !!notify_user },
```

Comment:
**[non-blocker]**

Why is the spreading part here needed ? Could just be notify_user: !!notify_user